README.md

```diff
@@ -157,6 +157,11 @@ is demonstrating.
   [Virtual Server Instances (VSIs)]((https://www.ibm.com/cloud/virtual-servers))
   in your IBM Cloud account and run workload on them.
 
+#### Function
+
+- [function](function)
+  This will show how to create Python and NodeJs functions with additional modules/packages
+
```

Comment:
Needs an updated link.

Reply:
Also can you change description to say "This example shows how to...."